app/main.ipynb

```diff
@@ -192,18 +244,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A.mean(axis=1)",
```

Comment:
The mean is calculated along axis 1, which gives the mean for each row. To find the mean for each column, use `axis=0`.